tests/keras/layers/test_core.py

```diff
@@ -187,9 +240,10 @@ def test_activity_regularization():
     z = core.Dense(2)(x)
     y = layer(z)
     model = Model(input=x, output=y)
-    model.compile('rmsprop', 'mse', mode='FAST_COMPILE')
+    model.compile('rmsprop', 'mse') # , mode='FAST_COMPILE')
```

Comment:
May have been something about my environment, it wasn't working when I tested it